src/index.js

```diff
@@ -9,10 +10,8 @@ const forge = require('node-forge')
 const protobuf = require('protocol-buffers')
 const path = require('path')
 
-const isNode = !global.window
-
 // protobuf read from file
-const messages = isNode ? protobuf(fs.readFileSync(path.resolve(__dirname, 'pb/crypto.proto'))) : protobuf(require('buffer!./pb/crypto.proto'))
+const messages = protobuf(fs.readFileSync(path.resolve(__dirname, '../protos/crypto.proto')))
```

Comment:
This seems to be braking the browser tests on `js-ipfs`. Why did you remove the `isNode` check?
cc @dignifiedquire @diasdavid